front/src/modules/trainschedule/components/ManageTrainSchedule/Allowances/Allowances.tsx

```diff
@@ -1,259 +0,0 @@
-    useState<OverlapAllowancesIndexesType>([false, false]);
-
-  const [engineeringOverlapAllowancesIndexes, setEngineeringOverlapAllowancesIndexes] =
-    useState<OverlapAllowancesIndexesType>([false, false]);
-
-  const standardAllowanceValue = useMemo(
-    () => getAllowanceValue(standardAllowance.default_value),
-    [standardAllowance]
-  );
-
-  const { updateAllowances } = useOsrdConfActions();
-
-  const setStandardDistribution = (distribution: StandardAllowance['distribution']) => {
-    setStandardAllowance({ ...standardAllowance, distribution });
-  };
-
-  const toggleStandardAllowanceSelectedIndex = (AllowanceIndex?: number) => {
-    setStandardAllowanceSelectedIndex(
-      AllowanceIndex !== standardAllowanceSelectedIndex ? AllowanceIndex : undefined
-    );
-  };
-  const toggleEngineeringAllowanceSelectedIndex = (AllowanceIndex?: number) => {
-    setEngineeringAllowanceSelectedIndex(
-      AllowanceIndex !== EngineeringAllowanceSelectedIndex ? AllowanceIndex : undefined
-    );
-  };
-
-  const resetFunction = (type: AllowancesTypes) => {
-    if (type === AllowancesTypes.standard) {
-      setStandardAllowance(initialStandardAllowance);
-      setStandardAllowanceSelectedIndex(undefined);
-    }
-    if (type === AllowancesTypes.standard) {
-      setEngineeringAllowances([]);
-      setEngineeringAllowanceSelectedIndex(undefined);
-    }
-  };
-
-  const updateStandardAllowanceDefaultValue = (newDefaultValue: AllowanceValueForm) => {
-    setStandardAllowance({ ...standardAllowance, default_value: newDefaultValue });
-  };
-
-  const updateStandardAllowances = (newStandardAllowanceRanges: RangeAllowanceForm[]) => {
-    setStandardAllowance({
-      ...standardAllowance,
-      ranges: newStandardAllowanceRanges,
-    });
-    setStandardAllowanceSelectedIndex(undefined);
-  };
-
-  const updateEngineeringAllowances = (newStandardAllowanceRanges: EngineeringAllowanceForm[]) => {
-    setEngineeringAllowances(newStandardAllowanceRanges);
-    setEngineeringAllowanceSelectedIndex(undefined);
-  };
-
-  useEffect(() => {
-    const newRanges = standardAllowance.ranges.map((allowance) =>
-      allowance.isDefault ? { ...allowance, value: standardAllowance.default_value } : allowance
-    );
-    setStandardAllowance({ ...standardAllowance, ranges: newRanges });
-  }, [standardAllowance.default_value]);
-
-  // dispatch only the valid allowances in the store
-  useEffect(() => {
-    const standardAllowanceDefaultValue = getAllowanceValue(standardAllowance.default_value);
-    const validStandardAllowance =
-      standardAllowanceDefaultValue && standardAllowanceDefaultValue > 0
-        ? [standardAllowance as StandardAllowance]
-        : [];
-    dispatch(
-      updateAllowances([...validStandardAllowance, ...engineeringAllowances] as Allowance[])
-    );
-  }, [standardAllowance, engineeringAllowances]);
-
-  return (
-    <div className="operational-studies-allowances">
```

Comment:
I think you can remove the css related to `operational-studies-allowances`

Reply:
Done (removed the css, it's also used in some e2e tests, but @Maymanaf wanted to keep the tests even if not used in case he would reuse some of them)